server/services/user.service.js

```diff
@@ -264,9 +264,18 @@ export const updateUser = async (updatedUser) => {
 
 export const getAllUsers = async ({ page = 1, limit = 10, ...query } = {}) => {
   const filterQuery = buildFilterQuery(USER_FILTERS, query);
+  let sortQuery;
+
+  const sortKeys = buildSortQuery(SORT_FILTERS, USER_FILTERS, query);
```

Comment:
```suggestion
    const sortQuery = buildSortQuery(USER_FILTERS, query);
```

Reply:
Now that the 2 functions have the same number of parameters, we can combine them together to make them easier to work with.

```suggestion
  const { filterQuery, sortQuery } = buildFilterAndSortQuery(USER_FILTERS, query);
```

Then in filters.js, you will define the new function as:

```js
export const buildFilterAndSortQuery = (allFilters, query) => ({
  filterQuery: buildFilterQuery(allFilters, query),
  sortQuery: buildSortQuery(allFilters, query),
});
```